app_flutter/lib/commit_box.dart

```diff
@@ -178,6 +174,6 @@ class CommitOverlayContents extends StatelessWidget {
 
   Future<void> _openGithub() async {
     final String githubUrl = 'https://github.com/${commit.repository}/commit/${commit.sha}';
-    launch(githubUrl);
+    await launch(githubUrl);
```

Comment:
Since this defers to the platform and doesn't impact the direct application, have you thought about using unawaited()?